srcpkgs/apache-jmeter/template

```diff
@@ -0,0 +1,42 @@
+# Template file for 'apache-jmeter'
+pkgname=apache-jmeter
+version=5.1.1
+revision=1
+hostmakedepends="openjdk apache-ant"
+depends="virtual?java-runtime"
+short_desc="Application to load test functional behavior and measure performance"
+maintainer="Kyle Nusbaum <knusbaum+void@sdf.org>"
+license="Apache-2.0"
+homepage="https://jmeter.apache.org/"
+distfiles="http://apache.osuosl.org//jmeter/source/apache-jmeter-${version}_src.tgz"
+checksum=133fee3ccc300c759b5fbba4dd870db8e69aa0f72eca080fb37a8fb56c6caf6b
+
+do_configure() {
+	ant download_jars
+}
+
+do_build() {
+	ant install
+}
+
+do_install() {
+	rm -Rf src test build
+
+	vlicense LICENSE
+	vmkdir /opt/apache-jmeter
```

Comment:
/opt is disallowed for package owned files, please pick a new directory.  Generally /usr/share/<pkgname> is a good choice.

Reply:
Not for "binary" files, `/usr/lib` or maybe even `/usr/libexec`.

Reply:
oh right, I forgot about that that the "compiled" artifacts will wind up in here.

Reply:
I'll put them in `/usr/libexec`.